ReactiveCocoa/Swift/EventLogger.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  EventLogger.swift
+//  ReactiveCocoa
+//
+//  Created by Rui Peres on 30/04/2016.
+//  Copyright © 2016 GitHub. All rights reserved.
+//
+
+import Foundation
+
+/// Used with the `debug` operator to customize
+/// the `Event` string.
+///
+/// This is useful, when you want to do more than
+/// just printing to the standard output.
+public protocol EventLogger {
```

Comment:
Since we don't have a clear use case in mind, I think we should remove `EventLogger` in the initial implementation. It adds complexity and it's much harder to remove API than to add it. (Removing is a breaking change; adding is a non-breaking change in this case.)

There's nothing that prevents anyone from implementation the behavior that this would allow.